### PR TITLE
rlp: updated ser/rlp/doc.go

### PR DESCRIPTION
**Pull request by ehnuje**
## Proposed changes

- updated ser/rlp/doc.go

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: ser/rlp/doc.go [hunk 1]
```diff
@@ -33,5 +33,13 @@ array).
 RLP values are distinguished by a type tag. The type tag precedes the
 value in the input stream and defines the size and kind of the bytes
 that follow.
+
+Source Files
+
+  - decode.go    : provides Decoder interface and implements decoding of default types
+  - encode.go    : provides Encoder interface and implements encoding of default types
+  - raw.go       : includes RawValue related operations, an encoded RLP value
```
Comment (kjhman21) [inline ser/rlp/doc.go]:
```suggestion
  - raw.go       : includes RawValue (an encoded RLP value) related operations
```

File: ser/rlp/doc.go [hunk 2]
```diff
@@ -33,5 +33,13 @@ array).
 RLP values are distinguished by a type tag. The type tag precedes the
 value in the input stream and defines the size and kind of the bytes
 that follow.
+
+Source Files
+
+  - decode.go    : provides Decoder interface and implements decoding of default types
+  - encode.go    : provides Encoder interface and implements encoding of default types
+  - raw.go       : includes RawValue (an encoded RLP value) related operations
+  - typecache.go : caches typeinfo which includes decoder and writer
```
Comment (KimKyungup) [inline ser/rlp/doc.go]: ?
```suggestion
  - typecache.go : cached typeinfo which includes decoder and writer
```

Comment (KimKyungup) [inline ser/rlp/doc.go]: ?
```suggestion
  - typecache.go : cached typeinfos which include decoder and writer
```

Comment (ehnuje) [reply ser/rlp/doc.go]: @KimKyungup I think `caches` as verb seems more appropriate, how do you think?

Comment (KimKyungup) [reply ser/rlp/doc.go]: I got it. :)

**Pull request merged**